ras/ras_lsvpd.py.data/ras_lsvpd.yaml

```diff
@@ -0,0 +1,6 @@
+run_type: !mux
+    upstream:
+        lsvpd_url: 'https://github.com/power-ras/lsvpd/archive/refs/heads/master.zip'
```

Comment:
same thing here too. we can remove the hard_coded url as we are already hard_coding as default_value

Reply:
sure, will update.